mxcube3/routes/Login.py

```diff
@@ -125,7 +125,6 @@ def loginInfo():
 
     login_info = login_info["loginRes"] if login_info is not None else {}
     login_info = limsutils.convert_to_dict(login_info)
-    limsutils.update_mxcube_session(login_id, login_info)
```

Comment:
It looks to me like some logic that once were in limsutils is now missing. The method update_mxcube_session made sure that the mxcube Session object was kept in sync when for instance the server is restarted.

Did you put it somewhere else ?, the data paths will not be correct if this information is not kept.